src/components/calendar/fCCalendar.tsx

```diff
@@ -37,35 +38,72 @@ export default function FCCalendar(props: CalendarData) {
     if (isDesktop) {
       return 'dayGridMonth'
     } else {
-      return 'dayGridDay'
+      return 'timeGridDay'
     }
   }
 
   return (
-    <Box className="rustic-fc-calendar">
+    <Box
+      className="rustic-fc-calendar"
```

Comment:
Have you tried configuring the [themesystem](https://fullcalendar.io/docs/themeSystem) ? 
I think we would get better styling using that if the Event display properties are insufficient.

Additional References - 
* https://fullcalendar.io/docs/event-display
* https://materialstyle.github.io/